rpm-testing/Vagrantfile

```diff
@@ -102,22 +92,39 @@ EOF'
       set -a -e -x
       source /etc/sysconfig/archivematica-dashboard
       cd /usr/share/archivematica/dashboard
-      /usr/lib/python2.7/archivematica/dashboard/bin/python manage.py syncdb --noinput
+      /usr/share/python/archivematica-dashboard/bin/python manage.py migrate --noinput
     ";
 
+    # Indexless
+    sudo sh -c 'echo "ARCHIVEMATICA_DASHBOARD_DASHBOARD_SEARCH_ENABLED=false" >> /etc/sysconfig/archivematica-dashboard'
+    sudo sh -c 'echo "ARCHIVEMATICA_MCPSERVER_MCPSERVER_SEARCH_ENABLED=false" >> /etc/sysconfig/archivematica-mcp-server'
+
     sudo -u root systemctl enable archivematica-mcp-server
     sudo -u root systemctl start archivematica-mcp-server
-    sudo -u root systemctl enable archivematica-mcp-client
-    sudo -u root systemctl start archivematica-mcp-client
     sudo -u root systemctl enable archivematica-dashboard
     sudo -u root systemctl start archivematica-dashboard
-    sudo -u root systemctl enable fits-nailgun
-    sudo -u root systemctl start fits-nailgun
     sudo -u root systemctl reload nginx
 
-    # TODO: configure clamav-daemon and freshclam
-    sudo cp /usr/bin/clamscan /usr/bin/clamdscan
-    sudo ln -sf /usr/bin/7za /usr/bin/7z
+    #
+    # MCPClient
+    #
+
+    sudo -u root rpm -Uvh https://forensics.cert.org/cert-forensics-tools-release-el7.rpm
+    sudo -u root rpm -Uvh https://li.nux.ro/download/nux/dextop/el7/x86_64/nux-dextop-release-0-5.el7.nux.noarch.rpm
+    sudo -u root yum install -y archivematica-mcp-client
+    sudo ln -s /usr/bin/7za /usr/bin/7z
+    sudo -u root sed -i 's/^#TCPSocket/TCPSocket/g' /etc/clamd.d/scan.conf
+    sudo -u root sed -i 's/^Example//g' /etc/clamd.d/scan.conf
+
+    # Indexless
+    sudo sh -c 'echo "ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_SEARCH_ENABLED=false" >> /etc/sysconfig/archivematica-mcp-client'
```

Comment:
As this vm is used for more than than indexless tests, can you use an env var to choose when will this be used?

Reply:
Ok, added. See commit `rpm/install: add SEARCH_ENABLED env`.